test/patest_dsound_find_best_latency_params.c

```diff
@@ -201,7 +202,7 @@ static void printWindowsVersionInfo( FILE *fp )
 
 
     fprintf( fp, "OS name and edition: %s %s\n", osName, osProductType );
-    fprintf( fp, "OS version: %d.%d.%d %S\n",
+    _ftprintf( fp, TEXT("OS version: %d.%d.%d %s\n"),
```

Comment:
Why "_fprintf"?

Reply:
Because `osProductType` comes from `GetVersionEx()` (as opposed to `GetVersionExA()` or `GetVersionExW()`), the string type is a [`TCHAR`](https://learn.microsoft.com/en-us/cpp/text/generic-text-mappings-in-tchar-h?view=msvc-170). The correct way to print a TCHAR is [`_tprintf`](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/printf-printf-l-wprintf-wprintf-l?view=msvc-170#generic-text-routine-mappings).

The corresponding warning is:

```
portaudio\test\patest_dsound_find_best_latency_params.c(205): warning C4477: 'fprintf' : format string '%S' requires an argument of type 'unsigned short *', but variadic argument 4 has type 'CHAR *'
```

It can be debated whether we should get rid of TCHAR code and switch to Unicode Win32 APIs everywhere, but I doubt we want this debate to be in scope for this PR, so for now I'm just making sure the code is at least consistent.